docs/high-availability/provision.sh

```diff
@@ -0,0 +1,179 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DISCOVERY=$1
+IP=`ip -o addr | grep 192 | cut -d' ' -f 7 | cut -d'/' -f 1` 
```

Comment:
its just a little hack to get the private ip of the running machine since only one 192 per machine.